local/network.go

```diff
@@ -603,10 +610,10 @@ func (ln *localNetwork) removeNode(nodeName string) error {
 	return nil
 }
 
-// Assumes [net.lock] is held
-func (ln *localNetwork) isStopped() bool {
+// Returns whether Stop has been called.
+func (ln *localNetwork) stopCalled() bool {
```

Comment:
I prefer `isStopping` or something like that, reflects more that it is in a certain state

Reply:
I thought about that, but then I didn't name it that because it might actually be _stopped_ as opposed to _stopping_